basxconnect/core/models/persons.py

```diff
@@ -188,6 +189,28 @@ def save(self, *args, **kwargs):
 
         self.default_sorting_name = self.name
 
+        if apps.is_installed("basxconnect.mailer_integration") and hasattr(
```

Comment:
Could this be a candidate for a signal handler? It might be a bit cleaner.

https://docs.djangoproject.com/en/dev/topics/signals/

Reply:
good point.

done